python/GafferImageUI/FormatPlugValueWidget.py

```diff
@@ -212,3 +203,20 @@ def __contextChanged( self, context, key ) :
 			self._requestUpdateFromValues()
 
 GafferUI.PlugValueWidget.registerType( GafferImage.FormatPlug, FormatPlugValueWidget )
+
+def _formatLabel( fmt, context ) :
+
+	if fmt == GafferImage.Format() :
+		return "Default ( %s )" % GafferImage.FormatPlug.getDefaultFormat( context )
+	else :
+		name = GafferImage.Format.name( fmt )
+		if name :
+			return "%s ( %s )" % ( name, str( fmt ) )
+		else :
+			return "Custom"
```

Comment:
What is `str( fmt )` producing here?  Is it the actual dimensions, ie 1920x1080?  Why show that for registered format but omit that when it's Custom?

Reply:
Yep, the dimensions. Good point - included dimensions with "Custom" in https://github.com/GafferHQ/gaffer/pull/5089/commits/5b330cd945b89397933b37b9ac18ca92307638b6.